packages/react-windows.1.2.0/opam

```diff
@@ -0,0 +1,30 @@
+opam-version: "1.2"
+maintainer: "Pierre Boutillier <pierre.boutillier@laposte.net>"
+homepage: "http://erratique.ch/software/react"
+authors: ["Daniel Bünzli <daniel.buenzl i@erratique.ch>"]
+doc: "http://erratique.ch/software/react/doc/React"
+#dev-repo: "http://erratique.ch/repos/react.git"
+#bug-reports: "https://github.com/dbuenzli/react/issues"
+tags: [ "reactive" "declarative" "signal" "event" "frp" "org:erratique" ]
+license: "BSD3"
+depends: [
+  "ocaml-windows"
+  "ocamlfind"
+  "ocamlbuild" {build}
+]
+build:
+[
+  [ "ocaml" "pkg/git.ml" ]
+  [ "env" "OCAMLFIND_TOOLCHAIN=windows"
+    "ocamlbuild" "-use-ocamlfind" "-pkg" "compiler-libs.toplevel"
+      "src/react.cma" "src/react_top.cma" "src/react.cmxa" "src/react_top.cmxa"
+      "src/react.cmxs" "src/react_top.cmxs" "pkg/META"
```

Comment:
... or this. Actually, you can simply copy react over from https://github.com/whitequark/opam-cross-android/tree/master/packages/react-android.1.2.0.